src/commands/createWebApp/WebAppCreateStep.ts

```diff
@@ -96,12 +96,27 @@ export class WebAppCreateStep extends AzureWizardExecuteStep<IWebAppWizardContex
                 default:
             }
         }
+        newSiteConfig.virtualApplications?.push({
+            virtualPath: '/',
+            physicalPath: 'site\\wwwroot',
+            preloadEnabled: true
+        })
+        // required property for setting "preloadEnabled" in VirtualApplications "true"
+        // "alwaysOn" cannot be true for "Free" tier
+        if (context?.plan?.sku?.tier !== 'Free') {
+            newSiteConfig.alwaysOn = true;
```

Comment:
Can I get more detail on "preloadEnabled" and "alwaysOn"? Without being super familiar with those settings, it sounds like it'd be a decent amount more expensive if my site is always running.

Reply:
From what I learned, both preloading and "alwaysOn" go together, and they are properties that enable faster loading times of the web app by keep the site/server alive even without usage. 
I am not sure about the specifics for Azure but I agree that it doesn't seem like a good idea to set them to `true` by default. 
Both "preloadEnabled" and "alwaysOn" seem to better fit frequent usage/production-level usage scenarios.

Reply:
Just talked to Matt offline, he is good with not setting these values.